.github/workflows/conductor-authorizer-tests.yml

```diff
@@ -139,6 +139,29 @@ jobs:
         confirmation_chain_length: 3
         max_txn_query: 5
         query_sleep_time: 5
+        ethereum_node_url: https://goerli.infura.io/v3/6141be73a15d47748af0dc14f53d57d7
+        EOF
+        cat <<\EOF > ~/.zcn/bridge.yaml
+        owner:
+          Password: 02289b9
```

Comment:
Let's not expose passwords in public repos, instead we should use github Secrets.